Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+MAINTAINER Jonas Finnemann Jensen <jopsen@gmail.com>
+
+RUN apt-get update -y
+RUN apt-get upgrade -y
+RUN apt-get install -y qemu dnsmasq-base build-essential liblz4-tool iptables
```

Comment:
I would run `apt-get autoremove` here as well, just for the sake of having a clean environment.